build/NuSpecs/WindowsAppSDK-Nuget-Native.targets

```diff
@@ -22,6 +22,13 @@
       <Implementation>Microsoft.WindowsAppSDK.dll</Implementation>
     </Reference>
   </ItemGroup>
+  
+    <ItemGroup>
+    <Reference Include="$(MSBuildThisFileDirectory)..\..\lib\native\Microsoft.Windows.System.winmd">
```

Comment:
>native [](http://example.com/codeflow?start=61&length=6)

please coordinate with @kythant as he is changing this.

Reply:
[https://github.com/microsoft/WindowsAppSDK/pull/1219](https://github.com/microsoft/WindowsAppSDK/pull/1219)